less/home-page.less

```diff
@@ -45,31 +85,118 @@
     padding: 50px 60px;
```

Comment:
The padding here makes the bottom portion of the page a bit narrower than the rest of the page. It looks like if you remove the left-right padding it aligns more neatly in both the desktop and mobile views.

Reply:
Just took another look at the design file and you're correct, it was supposed to look that way, I was wrong!